sys/net/gnrc/link_layer/slip/gnrc_slip.c

```diff
@@ -185,6 +185,11 @@ static void _slip_send(gnrc_slip_dev_t *dev, gnrc_pktsnip_t *pkt)
     gnrc_pktbuf_release(pkt);
 }
 
+static inline int _slip_get(gnrc_netapi_opt_t *opt)
+{
+    return (opt->opt == NETOPT_IS_WIRED) ? 1 : -ENOTSUP;
```

Comment:
please expand to a multiline `if`, it makes it clearer to read and easier to expand this further in the future if we want to add another netopt to slip.